### ISSUE TITLE
[MAS1.4.3] [Visual Requirements -Live Chat] Luminosity Ratio is less than 4.5:1 for “Activity Logs and Learn more” elements

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1754]
**Actual Result:**
Luminosity Ratio is less than 4.5:1 for “Activity Logs and Learn more” elements 

**Expected Result:**
Luminosity Ratio should be greater than or equal to 4.5:1 for “Activity Logs and Learn more” elements 

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened
3.Navigate to the open bot and select it  
4. Dialog gest opened and connect the open bot  
5. Live chat screen gets opened 
6.Navigate to the Learn more and Activity logs right side of the pane.
7.Open 'Colour Contrast Analyser' application.
8. Verify whether Luminosity Ratio is less than 4.5:1 for Activity Logs and Learn more” elements 

**User Impact:**
Users with low vision or colour blindness needs that difference between contrast of text is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Tool: Colour contrast analyser 

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)
![MAS1 4 3_Live Chat](https://user-images.githubusercontent.com/53518994/63340605-1ac8ef00-c365-11e9-9c5e-d875a3531dcc.png)

**Comment by Kumar2608** [comment]
Verified in latest bot emulator Version 4.5.2-78641 and issue got fixed Hence closing this bug
Now Luminosity Ratio 4.5:1 for Learn more and Activity Logs
